README.md

```diff
@@ -25,6 +25,10 @@ Name | Description
 --- | ---
 [cisco.iosxr.iosxr](https://github.com/ansible-collections/cisco.iosxr/blob/main/docs/cisco.iosxr.iosxr_cliconf.rst)|Use iosxr cliconf to run command on Cisco IOS XR platform
 
+### Filter plugins
```

Comment:
filter/ and inventory/ dirs got generated by cli_rm_builder, but are actually empty. We can remove these sections.

Reply:
yes that make sense